lookout/style/format/rules.py

```diff
@@ -291,6 +295,11 @@ def base_model(self, value: Union[DecisionTreeClassifier, RandomForestClassifier
             check_is_fitted(value, "tree_")
         elif isinstance(value, RandomForestClassifier):
             check_is_fitted(value, "estimators_")
+        if self.prune_branches_algorithm == "reduced-error" and \
+                isinstance(value, RandomForestClassifier):
+            raise ValueError("A reduced-error pruning method can be used only for "
```

Comment:
This is fine: you iterate over estimators and prune them independently.

Reply:
but as I remember from your experiment, it makes things worse. 

Should I include it anyway?

Reply:
Yeah, please include this even though it works bad.